src/V3Options.cpp

```diff
@@ -1303,6 +1303,10 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
     DECL_OPTION("-fdead-assigns", FOnOff, &m_fDeadAssigns);
     DECL_OPTION("-fdead-cells", FOnOff, &m_fDeadCells);
     DECL_OPTION("-fexpand", FOnOff, &m_fExpand);
+    DECL_OPTION("-ffunc-opt", CbFOnOff, [this](bool flag) {  //
+        m_fFuncSplitCat = flag;
```

Comment:
I'd think this should instead (or additionally) set m_fFuncOpt (or m_fFunc as you named it - seems inconsistent now), and have that variable in the .h, then we are ready to have additional optimizations in V3FuncOpt.

Reply:
See #5602 which adds -ffunc-opt-balance-cat. -ffunc-opt controls both.